html/webappapis/system-state-and-capabilities/the-navigator-object/protocol.https.html

```diff
@@ -135,23 +146,22 @@
   'web+dots.are.forbidden',
   'web+dashes-are-forbidden',
   'web+digits123areforbidden',
-  'web+UpperCasesAreForbidden',
-];
-for (const scheme of denylist) {
+].forEach(scheme => {
   test(() => {
-    assert_throws_dom('SECURITY_ERR', () => { navigator.registerProtocolHandler(scheme, location.href + '/%s', 'foo'); });
+    // https://test:test/ does not parse and does not contain %s, but the scheme check happens first
+    assert_throws_dom('SECURITY_ERR', () => { navigator.registerProtocolHandler(scheme, 'https://test:test/', 'foo'); });
```

Comment:
You should also test a valid URL that does not contain %s, since those are two separate checks.

Reply:
That's already handled under invalid URLs. I can clean this up a bit though.